cmake/packaging/windows/InstallRedistributables.nsh

```diff
@@ -14,114 +14,43 @@
 
 !include x64.nsh
 
-Var X64VS2013Needed
-Var I86VS2013Needed
-Var X64VS2015Needed
-Var I86VS2015Needed
-
 Function InstallRedistributables
 
-    StrCpy $X64VS2013Needed "1"
-    StrCpy $I86VS2013Needed "1"
-    StrCpy $X64VS2015Needed "1"
-    StrCpy $I86VS2015Needed "1"
-
-    # Check if it is necessary to install to x64VS2013
+    # Check if it is necessary to install to redistributables.
```

Comment:
This file is repited. Remove it.

Reply:
Remove this file and other unnecessary files.